aws/resource_aws_elasticache_cluster.go

```diff
@@ -515,6 +521,12 @@ func resourceAwsElasticacheClusterUpdate(d *schema.ResourceData, meta interface{
 		}
 	}
 
+	if requestRecreate {
+		log.Printf("[DEBUG] Recreating ElastiCache Cluster (%s), opts:\n%s", d.Id(), req)
+		resourceAwsElasticacheClusterDelete(d, meta)
+		resourceAwsElasticacheClusterCreate(d, meta)
+	}
```

Comment:
Pretty crude, not sure if that will cut it, maybe I should add the return here, I think create returns the read function so maybe `return resourceAwsElasticacheClusterCreate(d, meta)`